.github/workflows/dodeploy.yml

```diff
@@ -1,21 +1,32 @@
 name: Sync to DigitalOcean Spaces
 
-on:
+on: 
   push:
-    branches:
+    branches: 
       - main
 
 jobs:
   deploy:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@master
+#     - name: Bump version number
```

Comment:
Commented code probably should be removed? If we need this in the future we can always copy from binder plugin's main.yml then

Reply:
Your call. IDK if this plugin will ever even need version numbers.